webapp/components/FilterMenu/FilterMenu.vue

```diff
@@ -1,32 +1,22 @@
 <template>
-  <ds-card class="filter-menu-card">
-    <ds-flex class="filter-menu-content">
-      <ds-flex-item>
-        <ds-heading size="h3">{{ $t('filter-menu.hashtag-search', { hashtag }) }}</ds-heading>
-      </ds-flex-item>
-      <ds-flex-item>
-        <div class="filter-menu-buttons">
-          <base-button
-            name="clear-search-button"
-            icon="close"
-            circle
-            @click="clearSearch"
-            v-tooltip="{
-              content: this.$t('filter-menu.clearSearch'),
-              placement: 'left',
-              delay: { show: 500 },
-            }"
-          />
-        </div>
-      </ds-flex-item>
-    </ds-flex>
-  </ds-card>
+  <base-card class="filter-menu">
+    <h2>{{ $t('filter-menu.hashtag-search', { hashtag }) }}</h2>
+    <base-button
+      icon="close"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-12T13:00:12Z" title="Wednesday, February 12th 2020, 2:00:12 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,32 +1,23 @@
 <template>
-  <ds-card class="filter-menu-card">
-    <ds-flex class="filter-menu-content">
-      <ds-flex-item>
-        <ds-heading size="h3">{{ $t('filter-menu.hashtag-search', { hashtag }) }}</ds-heading>
-      </ds-flex-item>
-      <ds-flex-item>
-        <div class="filter-menu-buttons">
-          <base-button
-            name="clear-search-button"
-            icon="close"
-            circle
-            @click="clearSearch"
-            v-tooltip="{
-              content: this.$t('filter-menu.clearSearch'),
-              placement: 'left',
-              delay: { show: 500 },
-            }"
-          />
-        </div>
-      </ds-flex-item>
-    </ds-flex>
-  </ds-card>
+  <base-card class="filter-menu">
+    <h2>{{ $t('filter-menu.hashtag-search', { hashtag }) }}</h2>
+    <base-button
+      name="clear-search-button"
```


```suggestion
      
```
This is no longer used, oder?
I think it was added for testing, but since you target `.base-card`, it should be removed.